tests/nodeos_extra_packed_data_test.py

```diff
@@ -78,16 +77,18 @@
         Print("Stand up cluster")
         specificExtraNodeosArgs = {}
         associatedNodeLabels = {}
-        if len(specificExtraNodeosArgs):
-            specificExtraNodeosArgs[prodCount - 2] = " --backing-store=rocksdb"
-            specificExtraNodeosArgs[prodCount - 1] = " --backing-store=rocksdb"
-
-        for i in range( int(prodCount / 2) ):
-            associatedNodeLabels[str(i)] = "209"
+        if pnodes > 1:
+            specificExtraNodeosArgs[pnodes - 1] = " --backing-store=rocksdb"
+        if pnodes > 3:
+            specificExtraNodeosArgs[pnodes - 2] = " --backing-store=rocksdb"
+
+        if alternateVersionLabelsFile is not None:
+            #that means we are in multiversion mode
+            for i in range( int(pnodes / 2) ):
+                associatedNodeLabels[str(i)] = "209"
         
-        if cluster.launch(totalNodes=prodCount, 
-                          prodCount=1, 
-                          onlyBios=onlyBios, 
+        if cluster.launch(totalNodes=pnodes, 
+                          pnodes=1,
```

Comment:
So do you only want 1 producer and all the rest just nodes that are processing?  I missed earlier since you were using prod_count I thought that was the number of producers you want.  If you only want that one processor, you shouldn't use "-p" above and instead you should use "-n" and set totalNodes=args.n above, and then use totalNodes throughout instead of pnodes.